src/vivarium/framework/time.py

```diff
@@ -139,18 +140,22 @@ def step_backward(self) -> None:
     def step_forward(self, index: pd.Index) -> None:
         """Advances the clock by the current step size, and updates aligned simulant clocks."""
         self._clock_time += self.step_size
-        if self.individual_clocks:
-            update_index = self.get_active_population(index, self.time)
-            pop_to_update = self.individual_clocks.get(update_index)
-            if not pop_to_update.empty:
-                pop_to_update["step_size"] = self.step_size_pipeline(update_index)
-                pop_to_update["next_event_time"] = self.time + pop_to_update["step_size"]
-                self.individual_clocks.update(pop_to_update)
+        if self._individual_clocks:
+            update_index = self.get_active_simulants(index, self.time)
+            simulant_clocks_to_update = self._individual_clocks.get(update_index)
```

Comment:
Nit: My OCD would drive me to come up with a variable name that would enable these to stay on one line, but it's obviously not necessary.